lms/static/js/bookmarks/views/bookmark_button.js

```diff
@@ -44,8 +42,9 @@
                         view.$el.trigger('bookmark:add');
                         view.setBookmarkState(true);
                     },
-                    error: function() {
-                        view.showError();
+                    error: function (jqXHR) {
+                        var response = jqXHR.responseText ? JSON.parse(jqXHR.responseText) : '';
+                        view.showError(response && response.user_message);
```

Comment:
If you moved `.user_message` to the previous line, it would make this logic easier to understand.